taskfiles/k8s.yml

```diff
@@ -310,3 +318,16 @@ tasks:
     cmds:
       - 'echo "~~~ Creating operator v2 helm e2e tests :k8s:"'
       - ./hack/v2-helm-setup.sh
+
+  package-and-publish-operator-chart:
+    cmds:
+      - defer: 'helm registry logout registry-1.docker.io'
+      - helm registry login registry-1.docker.io -u {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
+      - helm dep update charts/operator
+      # To change name of the operator helm chart malformer program is executed
+      - go run ./nightly-malformer/...
```

Comment:
It looks like you could accomplish this by just running `helm package --version $TAG_NAME`

Reply:
The most important change is changing the Chart name just to match the https://hub.docker.com/r/redpandadata/redpanda-operator-nightly docker hub repository. That way we can host helm chart and containers under one repository.

Reply:
Not to be too crotchety but it looks like this could also be handled by a tiny bit of scripting with `tar` and `yq`. Doing so would be more easily reusable for other charts and not dependent on go loading `Chart.yaml`.